Pod/Classes/BONChain.h

```diff
@@ -119,14 +119,16 @@ typedef BONChain *BONCNonnull (^BONTagComplexStyles)(BONGeneric(NSArray, BONTag
 
 // concatenation
 - (void)appendLink:(id<BONTextable>)link;
-- (void)appendLink:(id<BONTextable>)link separator:(BONNullable NSString *)separator;
+- (void)appendLink:(id<BONTextable>)link separatorTextable:(BONNullable id<BONTextable>)separator;
 
 @end
 
 @interface BONChain (Deprecated)
 
 @property (copy, nonatomic, readonly) BONChainColor textColor __attribute__((deprecated("use -color instead")));
 
+- (void)appendLink:(id<BONTextable>)link separator:(BONNullable NSString *)separator __attribute__((deprecated("use -appendLink: separatorTextable: instead")));
```

Comment:
#req no space: `-appendLink:separatorTextable:`. That wasn't a typo - it's just a quirk of how Obj-C selector names are spelled.